pandas/core/frame.py

```diff
@@ -1984,7 +1984,10 @@ def to_parquet(self, fname, engine='auto', compression='snappy',
         Parameters
         ----------
         fname : str
-            String file path.
+            File path or Root Directory path. Will be used as Root Directory
+            path while writing a partitioned dataset.
+
+            .. versionchanged:: 0.24.0
```

Comment:
Does this render? Might need a blank space after the directive?

Reply:
Looks like it does not.

Documentation shows API Reference for DataFrame.to_parquet, read_parquet() for parquet.py and io.rst, which explains about reading and writing to parquet.

Reply:
Wasn't referring to the public API as much as the docstring in isolation. If you can ensure there is a blank line after this and any of the other `versionadded` directives should be the last thing

Reply:
have added an empty line after all versionchanged and versionadded. Hope you can have a look